owning-a-smart-contract.md

```diff
@@ -0,0 +1,423 @@
+# TUTORIAL: Sophia basics - owning a smart contract and owner privileges
+
+## Tutorial Overview
+This tutorial will walk teach you how one can access restrict a method to the deployer of the smart contract itself. This is a useful tool for defining critical business logic.
+## Prerequisites
+- Installed Aeproject - For installation steps reffer to [this tutorial](smart-contract-deployment-in-aeproject.md)
+
+## 1. Prepare your environment
+
+Before we begin coding up we should start by initializing an aepp project.
+### 1.1 aeproject init
+
+Let's create a new empty folder and initialise an aepp project.
+
+```
+mkdir ownable-project
+cd ownable-project
+aeproject init
+```
+
+### 1.2 aeproject epoch
+Another thing we would require is a local working environment to develop against. Luckily `aeproject` gives us that out of the box. Just type in the following command
+
+```
+aeproject node
```

Comment:
when I install the aeproject from the default `develop` branch, I still don't have the `node` command

Reply:
Also to use the command, `docker` and `docker-compose` need to be installed, which is not explained in the referenced tutorial https://github.com/aeternity/tutorials/blob/5754bd471e7d5a283fbe99a8d2d503076701f813/smart-contract-deployment-in-aeproject.md

Reply:
aeproject PR is now merged